docs/source/query/overview.rst

```diff
@@ -0,0 +1,216 @@
+==============
+Querying Snuba
+==============
+
+This guide drives you through the process of authoring and testing a
+Snuba query.
+
+Exploring the Snuba data model
+==============================
+
+In order to architect a Snuba query, the first step is being able to
+know which Dataset you should query, which Entities you should select
+and what the schema of each Entity is.
+
+For an introduction about Datasets and entities, see the :doc:`/architecture/datamodel`
+section.
+
+Unfortunately at present there isn't an easy to browse description of
+the data model, so this will be annoying. (a tool to explore the data
+model will eventually be built).
+
+Datasets can be find `in this module <https://github.com/getsentry/snuba/blob/master/snuba/datasets/factory.py>`_.
+Each dataset is a class that references the Entities.
+
+Entity classes are `here <https://github.com/getsentry/snuba/tree/master/snuba/datasets/entities>`_.
+Once you found the Entity (or Entities) you want to query, you will need
+to know the schema of each of them.
+
+Entity schemas are defined as a `ColumnSet <https://github.com/getsentry/snuba/blob/master/snuba/clickhouse/columns.py#L472>`_.
+Each Entity declares one ColumnSet which can be find in the Entity class as
+`abstract_column_set`. An example is in the `Transaction <https://github.com/getsentry/snuba/blob/master/snuba/datasets/entities/transactions.py#L99) for transactions. The transactions Entity relies on the schema of its main storage, which is [here](https://github.com/getsentry/snuba/blob/master/snuba/datasets/storages/transactions.py#L29>`_
+entity.
+
+Example: If we wanted a the count and the average of transactions duration
```

Comment:
I was expecting an actual full example or query snippet here. As it is currently, I don't think this sentence says much and we could probably do without it.

Reply:
I removed the paragraph. What I want to point out was how to find the column, not necessarily how to build the query.
I think all this will be less confusing when we will have a tool to navigate the data model